### PR TITLE
Create ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61415](https://cumulocity.atlassian.net/browse/MTM-61415)
Original PR: [7463](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7463)

[MTM-61415]: https://cumulocity.atlassian.net/browse/MTM-61415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix: do not remove scripts from the svg in the scada widget when none option is set (#7397) (#7463)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md]: The Claude AI suggested to use the following title:
```suggestion
title: script tags are preserved in SCADA widget when 'none' option is selected
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61415
+version: 1021.0.2
+---
+fix: do not remove scripts from the svg in the scada widget when none option is set (#7397) (#7463)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md]: The Claude AI suggested to use the following body:
```suggestion
SCADA widget preserves script tags when 'none' option is selected.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by rahulc8y — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61415
+version: 1021.0.2
+---
+SCADA widget preserves script tags when 'none' option is selected.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md]:
```suggestion
The SCADA widget now preserves script tags if the code sanitization option is set to "none".
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: script tags are preserved in SCADA widget when 'none' option is selected
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-2-fix-do-not-remove-scripts-from-the-svg-in-the-scada.md]:
```suggestion
title: Script tags are preserved in SCADA widgets
```

**Review by DawidSAG — APPROVED** [review]

**Pull request merged**